tests/foreman/cli/test_satellitesync.py

```diff
@@ -588,28 +597,31 @@ def test_positive_export_format_inheritance(
         :expectedresults:
             1. Incremental exports inherit the format from the previous complete export.
             2. When using --from-history-id, the format is inherited from the specified export.
+            3. Tests work with and without destination-server parameter.
 
-        :Verifies: SAT-32715, SAT-38392
+        :Verifies: SAT-32715, SAT-38691
 
-        :BlockedBy: SAT-38392
+        :BlockedBy: SAT-38691
 
         :customerscenario: true
         """
         if subject == 'library':
             cpl_export = target_sat.cli.ContentExport.completeLibrary
             inc_export = target_sat.cli.ContentExport.incrementalLibrary
             exp_params = {'organization-id': function_org.id}
-            importable_msg = f'{function_org.name}/Export-Library'
-            syncable_msg = f'{function_org.name}/Export-Library-SYNCABLE'
         elif subject == 'repository':
             cpl_export = target_sat.cli.ContentExport.completeRepository
             inc_export = target_sat.cli.ContentExport.incrementalRepository
             exp_params = {'id': function_synced_custom_repo['id']}
-            importable_msg = f'{function_org.name}/Export-{function_synced_custom_repo.name}'
-            syncable_msg = f'{function_org.name}/Export-SYNCABLE-{function_synced_custom_repo.name}'
         else:
             raise ValueError('Unsupported export subject used in parametrization')
 
+        # Add destination-server parameter if testing with it
+        # Note: destination-server is just a human-readable label, not an actual server
+        # Repository exports don't support destination-server (handled by parameterization)
+        if with_destination_server:
+            exp_params['destination-server'] = 'test-destination-server'
```

Comment:
I'd move this part under `if subject == 'library':` since it applies only to the "library exports".
There is yet another reason I'll explain in the next comment.